src/sass/components/_path.scss

```
@@ -0,0 +1,71 @@
+// Path Overrides
+// ==============
+
+// redefine path-specific color variables
+$color-text-path-current: $color-text-default;
+$color-background-path-active: $brand-background-primary;
+$color-background-path-active-hover: $color-brand-darker;
+$color-background-path-complete: $color-background-path-incomplete-hover;
+$color-background-path-complete-hover: $color-gray-6;
+
+.slds-path__item {
+  min-width: 0;
```

```suggestion
  min-width: 3em;
```

I think we need something here to ensure they don't disappear. might also be good to do something so that *current step* exapands more than others…